includes/Types.php

```diff
@@ -24,14 +24,14 @@
  * limitations under the License.
  */
 
-namespace Google\Web_Stories\Traits;
+namespace Google\Web_Stories;
```

Comment:
Should this class live in the media namespace?

Reply:
Makes sense to me.

Makes it clearer that it's about "media types" and not other types of "types".

Reply:
Done!